examples/cisco/demo_trm.pp

```diff
@@ -0,0 +1,54 @@
+# Manifest to demo TRM providers
+#
+# Copyright (c) 2018 Cisco and/or its affiliates.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+class ciscopuppet::cisco::demo_trm {
+
+  if platform_get() =~ /n9k/ {
```

Comment:
Not clear on how this indicates the platform supports TRM?

Reply:
I've updated this to add another check for image as well.